cypress/support/fragments/settings/calendar/library-hours/library-hours.js

```diff
@@ -87,4 +93,14 @@ export default {
 
     cy.do(Button({ id: 'clickable-delete-confirmation-confirm' }).click());
   },
+
+  checkDeleteButtonAbsence() {
+    cy.expect(Button(buttonLabels.delete).absent());
+  },
+
+  clearCreatedEvents() {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    this.openCalendarEvents();
+    this.deleteCalendarEvent();
```

Comment:
I suggest to make this action trough API, because you use this function inside "after"
Login is very long action

https://s3.amazonaws.com/foliodocs/api/mod-calendar/r/calendar.html#calendar_periods__servicepointid__period__periodid__delete

Reply:
@MaksimTerekhov Unfortunately, it is impossible to do it through API.
To delete calendar event I need to know event id. But it is unknown because it is created through user interface.

Reply:
Also, I have found a similar approach here - https://github.com/folio-org/stripes-testing/blob/d0aeb00c5166a1340d17aeacbf289b96f1870cfb/cypress/integration/checkin/checkin.test.spec.js#L39

and relied on it